src/setup.sh

```diff
@@ -273,7 +288,34 @@ function get_env_file {
 }
 
 #
-# Checks that this script can be run on the current machine and exits with an 
+# Checks if HomeBrew is installed.
+#
+function is_brew_installed {
+  type brew > /dev/null 2>&1
+}
+
+#
+# Checks if MacPorts is installed.
+#
+function is_port_installed {
+  type port > /dev/null 2>&1
+}
+
+#
+# Checks if all MacPorts requirements are met.
+#
+function check_port_prerequisites {
+  if ! type boot2docker > /dev/null 2>&1 ; then
```

Comment:
Boot2Docker is one of the items in the `port_install` list. Why is it a prerequisite?